cluster/kube/gateway/httproute_test.go

```
@@ -206,3 +206,38 @@ func TestCreateOrUpdateHTTPRouteNewRoutePlaceholderNotRoutable(t *testing.T) {
+	ctx := context.Background()
+
+	require.NoError(t, CreateOrUpdateHTTPRoute(ctx, dc, routeConfig(), routeDirective(), NoopHTTPRouteObserver{}))
+
+	ns := builder.LidNS(mtypes.LeaseID{})
+	labels := builder.AppendLeaseLabels(mtypes.LeaseID{}, map[string]string{builder.AkashManagedLabelName: "true"})
+	lm := make(map[string]interface{}, len(labels))
+	for k, v := range labels {
+		lm[k] = v
+	}
+	placeholder := &unstructured.Unstructured{Object: map[string]interface{}{
+		"apiVersion": "gateway.networking.k8s.io/v1",
```

golangci-lint (2.12.2)

[error] string `gateway.networking.k8s.io/v1` has 3 occurrences, make it a constant

(goconst)